### PR TITLE
Fixed markdown in tabs.

### PR DESCRIPTION
**Pull request by Shubham82**
This PR fixed the markdown in tabs that are not working under [Install using native package management](https://kubernetes.io/docs/tasks/tools/install-kubectl-linux/#install-using-native-package-management) 

Fixes: #35482

### CODE REVIEW

**Comment by Shubham82** [comment]
PTAL

**Review by Babapool — COMMENTED** [review]
The new changes LGTM. Thanks.
( This is an unofficial lgtm from my side-I am not an official SIG-Docs approver/reviewer)

**Comment by Shubham82** [comment]
@Babapool thanks for taking look at this!

File: content/en/docs/tasks/tools/install-kubectl-linux.md [hunk 1]
```diff
@@ -112,16 +112,13 @@ For example, to download version {{< param "fullversion" >}} on Linux, type:
...
-    ```
-   
-   {{< /note >}}
+{{< note >}}
+ If you use Debian 9 (stretch) or earlier you would also need to install `apt-transport-https`:
+```shell
+sudo apt-get install -y apt-transport-https
+```
+
+{{< /note >}}
```
Comment (tengqm) [inline content/en/docs/tasks/tools/install-kubectl-linux.md:121]: I don't think we really need such a complicated layout for these contents, i.e. a tab -> numbered list -> a note -> a code snippet.

Why don't we just remove the `{{< note >}}` shortcode? The "If you use ..." paragraph needs to be indented by three spaces to align with contents on line 109 then.

Comment (Shubham82) [reply content/en/docs/tasks/tools/install-kubectl-linux.md:121]: thanks, @tengqm for taking look at this.
It seems good to me. I will remove the  `{{< note >}}` shortcode, and align "If you use ..."  with the first point(line 109).

Will update it soon.

Comment (Shubham82) [reply content/en/docs/tasks/tools/install-kubectl-linux.md]: I have addressed the comment.
PTAL!

**Comment by tengqm** [comment]
Preview: https://deploy-preview-35504--kubernetes-io-main-staging.netlify.app/docs/tasks/tools/install-kubectl-linux/

**Pull request merged**